src/pages/Watchlist/Watchlist.js

```diff
@@ -71,7 +71,7 @@ const WatchlistPage = ({ location, history, watchlist }) => {
     }
   }, [watchlist.listItems])
 
-  const refetchAssets = () => {
+  const refetchAssets = (callback) => {
```

Comment:
Can we make a more specific name than 'callback' ?

Reply:
renamed to `onRefetchDone`